src/_plugins/release_registry.rb

```diff
@@ -68,8 +68,21 @@ def render(context)
       get_info["repo_url"]
     end
   end
+
+  class SdkCdnChecksumTag < ReleaseRegistryTag
+    def entity
+      "sdks"
+    end
+
+    def render(context)
+      file = @params[2]
+      algorithm = @params[3] || "sha384-base64"
+      get_info["files"][file]["checksums"][algorithm]
```

Comment:
I'd suggest we raise a RuntimeError here when the checksum is empty or doesn't exist (for any reason).
Otherwise, we can end up with broken integrity attribute.